### PR TITLE
Prepare 0.15.0 release

### PR DESCRIPTION
**Pull request by joshuagl**
Fixes N/A

**Description of the changes being introduced by the pull request**:

Update docs/CHANGELOG.md and bump version number for a 0.15.0 release

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Review by jku — APPROVED** [review]

**Comment by jku** [comment]
tox -e with-sslib-master:
> ERROR: Double requirement given: securesystemslib[crypto,pynacl]==0.17.0 (from -r /home/travis/build/theupdateframework/tuf/requirements-pinned.txt (line 11)) (already in securesystemslib[crypto,pynacl] from git+http://github.com/secure-systems-lab/securesystemslib.git@master#egg=securesystemslib[crypto,pynacl], name='securesystemslib')

not related to this PR: just noticed it here. It already happened before the sslib 0.17.0 release as well.

This seems to happen because requirements-test.txt contains `-r requirements-pinned.txt` which contains `securesystemslib[crypto,pynacl]==0.17.0` and pip seems to think that is not compatible with the git requirement... but how did that work before then: nothing important seems to have changed

**Comment by jku** [comment]
> tox -e with-sslib-master:
> > ERROR: Double requirement given: securesystemslib[crypto,pynacl]==0.17.0 (from -r /home/travis/build/theupdateframework/tuf/requirements-pinned.txt (line 11)) (already in securesystemslib[crypto,pynacl] from git+[http://github.com/secure-systems-lab/securesystemslib.git@master#egg=securesystemslib[crypto,pynacl]](https://github.com/secure-systems-lab/securesystemslib.git@master#egg=securesystemslib%5Bcrypto,pynacl%5D), name='securesystemslib')

Bisect tells me the problem is eb00d1434 . I'll file an issue. I think this does not have to necessarily block the release: it looks like a CI configuration issue

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## v0.15.0
+### Added
+* Simple TUF role metadata model in the `tuf.api` package for interacting with
+  metadata files directly, per-file without the overheads of reading and
+  writing the entire repository at once (#1112)
```
Comment (lukpueh) [inline docs/CHANGELOG.md]:
```suggestion
  writing the entire repository at once (#1112, ##1177)
```

**Comment by lukpueh** [comment]
Just took the liberty to add few items to the changelog, I hope that works for @joshuagl. @jku, would you mind taking another quick glance at it?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+  securesystemslib (#1183)
+* Overhauled logging to be less verbose and less alarming, by removing logging
+  in the library when an exception is raised (including the same information
+  that was logged) and using more appropriate log levels (#1145)
+* Make test output more useful by reducing and improving logging (#1145, #1104, #1170)
+* Make the `targets_path`, `metadata_path` and `confined_target_dirs` fields in
+  `tuf.client.updater`s mirror configuration optional (#1153, #1166)
+* Include LICENSE files with source distributions (#1162)
+* Update Python version to be used in release instructions (#1163)
+* Remove direct use of `coloroma` and dependency (#1180)
```
Comment (jku) [inline docs/CHANGELOG.md]:
```suggestion
* Remove direct use of `colorama` and dependency (#1180)
```

**Review by jku — APPROVED** [review]
LGTM, left one typo fix

File: docs/CHANGELOG.md (hunk 2 above)
Comment (lukpueh) [reply docs/CHANGELOG.md]: Great find! Thanks.

**Pull request merged**